docs/user/tutorials/01-10-mtls-authentication/configure-mtls-Gardener-certs.md

```diff
@@ -0,0 +1,547 @@
+# Configure mTLS Authentication Using Gardener-Managed Certificates
+Learn how to configure mutual TLS (mTLS) in SAP BTP, Kyma runtime using Gardener-managed Let's Encrypt server certificates and client certificates that you supply.
+
+## Context
+mTLS (mutual TLS) provides two‑way authentication: the client verifies the server's identity and the server verifies the client's identity. To enforce this authentication, the mTLS Gateway requires the following values: 
+- the server private key
+- the server certificate chain (server certificate plus any intermediate CAs)
+- the client root CA used to validate presented client certificates. 
+Each client connecting through the mTLS Gateway must have a valid client certificate and key and trust the server's root CA.
+
+In this procedure, Gardener’s Certificate resource requests a publicly trusted server certificate from Let’s Encrypt and creates a Secret that stores the certificate and private key.
+
+Because Gardener manages only the server certificate and key, you must supply the client root CA and create yet another Secret storing this value. Moreover, all clients interacting with the server (Kyma workloads) must trust the server's root CA (Let's Encrypt) and have their respective client certificate chains installed. This procedure uses self-signed client certificates, but for production use, it's strongly advised to use certificates issued by a trusted CA instead.
```

Comment:
```suggestion
Because Gardener manages only the server certificate and key, you must supply the client root CA and create yet another Secret storing this value. Moreover, all clients interacting with the server must trust the server's root CA (Let's Encrypt) and have their respective client certificate chains installed. This procedure uses self-signed client certificates, but for production use, it's strongly advised to use certificates issued by a trusted CA instead.
```

Maybe that should be "Istio Gateway" if specifying that here is needed?